File: openapi-generator/csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/CargoResponseSensors.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class CargoResponseSensors {
    /// <summary>
    /// Flag indicating whether the current cargo is empty or loaded.
    /// </summary>
    /// <value>Flag indicating whether the current cargo is empty or loaded.</value>
    [DataMember(Name="cargoEmpty", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cargoEmpty")]
    public bool? CargoEmpty { get; set; }

    /// <summary>
    /// Name of the sensor.
    /// </summary>
    /// <value>Name of the sensor.</value>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// ID of the sensor.
    /// </summary>
    /// <value>ID of the sensor.</value>
    [DataMember(Name="id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "id")]
    public long? Id { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class CargoResponseSensors {\n");
      sb.Append("  CargoEmpty: ").Append(CargoEmpty).Append("\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: openapi-generator/dart/lib/model/cargo_response_sensors.dart
part of openapi.api;

class CargoResponseSensors {
  /* Flag indicating whether the current cargo is empty or loaded. */
  bool cargoEmpty = null;
  /* Name of the sensor. */
  String name = null;
  /* ID of the sensor. */
  int id = null;
  CargoResponseSensors();

  @override
  String toString() {
    return 'CargoResponseSensors[cargoEmpty=$cargoEmpty, name=$name, id=$id, ]';
  }

  CargoResponseSensors.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['cargoEmpty'] == null) {
      cargoEmpty = null;
    } else {
          cargoEmpty = json['cargoEmpty'];
    }
    if (json['name'] == null) {
      name = null;
    } else {
          name = json['name'];
    }
    if (json['id'] == null) {
      id = null;
    } else {
          id = json['id'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'cargoEmpty': cargoEmpty,
      'name': name,
      'id': id
    };
  }

  static List<CargoResponseSensors> listFromJson(List<dynamic> json) {
    return json == null ? new List<CargoResponseSensors>() : json.map((value) => new CargoResponseSensors.fromJson(value)).toList();
  }

  static Map<String, CargoResponseSensors> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, CargoResponseSensors>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new CargoResponseSensors.fromJson(value));
    }
    return map;
  }
}

